app/models/user.rb

```diff
@@ -14,6 +14,8 @@ class User < ApplicationRecord
   before_update { ensure_not_depot_admin('update') } if :depot_admin?
   before_destroy { ensure_not_depot_admin('delete') } if :depot_admin?
 
+  has_many :orders
```

Comment:
what will happen to orders when a user is destroyed.
Don't allow to delete a user if its associated orders are present